winrt/test.internal/graphics/CanvasFontFaceUnitTests.cpp

```diff
@@ -180,6 +188,10 @@ TEST_CLASS(CanvasFontFaceTests)
 
         Assert::AreEqual(RO_E_CLOSED, canvasFontFace->GetGlyphIndices(1, &u, &u, &iPointer));
 
+        Assert::AreEqual(E_INVALIDARG, canvasFontFace->GetGlyphMetrics(1, &i, true, &u, &rectPointer));
```

Comment:
Were the unit tests able to run without error? It looks like this test will not pass- it should check against RO_E_CLOSED.